inspection/background.js

```diff
@@ -126,23 +120,21 @@ function loadListener(url) {
  */
 function createOpenURLEntry(url) {
 	if (url) {
-		chrome.tabs.create({url: url}, function (tab) {
+		chrome.tabs.create({ url: url }, function (tab) {
 			printLog('log', 'OPEN NEW PAGE');
-			localStore?.set({inspectingTab: JSON.parse(JSON.stringify(tab))}, () => {});
-			// todo: is `inspectingTab` a global var or local?
+			localStore?.set({ inspectingTab: JSON.parse(JSON.stringify(tab)) }, () => { });
 			printLog(inspectingTab);
```

Comment:
now we've removed `inspectingTab`, this line should also be removed?

Reply:
Done with this change and pushed it to remote